core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -5361,52 +5362,92 @@ private void checkLiteral2(String expression, String expected) {
 
   /** Test case for
    * <a href="https://issues.apache.org/jira/browse/CALCITE-6949">[CALCITE-6949]
-   * ClickHouse not support floor date to SECOND/MILLISECOND/MICROSECOND/NANOSECOND</a>. */
+   * ClickHouse not support floor date to SECOND/MILLISECOND/MICROSECOND/NANOSECOND</a>,
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6988">[CALCITE-6988]
+   * DuckDB dialect implementation</a>.
+   * */
   @Test void testFloorClickHouse() {
```

Comment:
Change the method name to testFloorXxx?

Reply:
+1, had changed the name to testFloorFunction. @xiedeyantu